src/molecules/formfields/GoogleAutoCompleteField/index.js

```diff
@@ -51,6 +51,21 @@ class GoogleAutoCompleteField extends Component {
   updateAddress = () => {
     const place = this.state.autocomplete.getPlace();
     const addressData = this.formatAddressInfo(place.address_components);
+
+    if (!addressData.number) {
+      const { value } = this.props.input;
+
+      if (value) {
+        const matches = value.match(/^(\d+)/);
```

Comment:
Also could something like `123 west 50th street` return `12350` instead?

Reply:
I'll make that first change, I like it! And no it shouldn't, because the regex will only capture sequential digits, when it hits the whitespace before `west` it'll stop. But I'll add a test for it just in case - it's good to have coverage there should anyone refactor this in the future!